### ISSUE TITLE
9.4.1.2/11.6 All checkboxes in the first column of the table use the "aria-label" attribute, which has been incorrectly applied in this case on the "span" element, which is not allowed. The "aria-label" attribute should be used together with the "input" element or, alternatively, an existing "label" element should be used instead of ARIA.

### ISSUE THREAD

**Issue by szaimen** [nextcloud/server#42238]
9.4.1.2/11.6 All checkboxes in the first column of the table use the "aria-label" attribute, which has been incorrectly applied in this case on the "span" element, which is not allowed. The "aria-label" attribute should be used together with the "input" element or, alternatively, an existing "label" element should be used instead of ARIA.

![image](https://github.com/nextcloud/server/assets/42591237/c265d217-369f-4a80-badd-b8e742afb4a8)

**Comment by JuliaKirschenheuter** [comment]
duplicate of https://github.com/nextcloud/server/issues/41894
